src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculate, Drawable {
+
+    protected String color;
+
+    public Figure() {
+    }
```

Comment:
```suggestion
    public Figure() {
    }
```

why do u need this constructor here?